heidegger_index/templates/work_detail.html

```diff
@@ -12,10 +12,32 @@ <h2>Index terms</h2>
 {% regroup page_refs by lemma as lemma_list %}
 <ul class="lemma-refs">
   {% for lemma, page_ref_list in lemma_list %}
```

Comment:
Als ik het goed begrijp verschijnen persoonslemma's en werklemma's nu dubbel op deze pagina: één keer in deze list en een keer in één van de gefilterde lists hierbeneden. Is dat gewenst?

Reply:
Nee, dat is niet het geval omdat ik alleen de lemmata onder 'index terms' weergeeft die geen gedefinieerd type hebben. 
https://github.com/andredelft/heidegger-index/blob/d3f34456cb1db61492d10e9419c6c72f1e007ed4/heidegger_index/templates/work_detail.html#L13-L15

Reply:
Ah ja, ik had dat if-statement gemist..